tests/integration_tests/functional/test_error_code.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests scenarios for Firecracker signal handling."""
+
+import os
+import time
+import platform
+import pytest
+
+
+@pytest.mark.skipif(
+    platform.machine() == "x86_64",
+    reason="We only test with a kernel for aarch64"
```

Comment:
`reason` should express why we skip the test for x86_64, and this is not clear.
why do we do this?

Reply:
can't we also trigger this behaviour on x86?

Reply:
As far as I know we can, but the behavior would be the same. The portion of code that is handling this error code does not care about architecture.

Reply:
This is a weak argument. We run many tests that should have the same behaviour across all platforms. Just because something should happen, doesn't mean it will.

Reply:
At the very least, create an issue for this and link it in the `reason`